tests/functional/spec/relationships.spec.js

```diff
@@ -72,6 +72,13 @@ describe('Relationships', function() {
           .getUrl().should.eventually.contain(ListCollectorPage.pageName);
       });
 
+      it.only('And go to the first relationship, Then the \'Brother or Sister\' option should have the text \'Including half brother or half sister\'', function() {
```

Comment:
You may want to remove this .only

Reply:
If only we had a linter for this stuff...